test/azure/terraform_azure_nsg_example_test.go

```diff
@@ -0,0 +1,46 @@
+// +build azure
+
+// NOTE: We use build tags to differentiate azure testing because we currently do not have azure access setup for
+// CircleCI.
+
+package test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/azure"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTerraformAzureNsgExample(t *testing.T) {
+	t.Parallel()
+
+	terraformOptions := &terraform.Options{
+		// The path to where our Terraform code is located
+		TerraformDir: "../../examples/azure/terraform-azure-nsg-example",
```

Comment:
Set `Vars` here and include `prefix`, including `random.UniqueId`: https://github.com/gruntwork-io/terratest/pull/626/files#r490168023

Reply:
Setting postfix variable from `random.UniqueId` in test code.